bpf/cpu/pyperf.h

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// Licensed under the Apache License, Version 2.0 (the "License")
+//
+// Copyright 2023 The Parca Authors
+
+#include "basic_types.h"
+#include "common.h"
+
+#define PYTHON_STACK_FRAMES_PER_PROG 16
+#define PYTHON_STACK_PROG_CNT 5
+#define MAX_STACK (PYTHON_STACK_FRAMES_PER_PROG * PYTHON_STACK_PROG_CNT)
+
+#define PYPERF_STACK_WALKING_PROGRAM_IDX 0
+
+typedef struct {
+  // u64 start_time;
+
+  // u64 interpreter_addr;
+  u64 thread_state_addr;
+
+  u32 py_version;
+} InterpreterInfo;
+
+enum python_stack_status {
+  STACK_COMPLETE = 0,
+  STACK_TRUNCATED = 1,
+  STACK_ERROR = 2,
+};
+
+typedef struct {
+  u64 timestamp;
+  u32 cpu;
```

Comment:
We don't seem to use this, do we need it?

Reply:
We can remove it. It's residue from sandbox app logic.

Reply:
Appreciate it! It's extra cycles that won't get any benefit :D